<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindReplaceDialog</class>
 <widget class="QDialog" name="FindReplaceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>150</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>430</width>
    <height>150</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>430</width>
    <height>150</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>查找/替换</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>411</width>
     <height>72</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QLineEdit" name="find_lineEdit"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="replace_label">
      <property name="text">
       <string>替换为：</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="replact_lineEdit"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="find_label">
      <property name="text">
       <string>查找内容：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QCheckBox" name="matchCase_cb">
      <property name="text">
       <string>区分大小写</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>158</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="replace_btn">
      <property name="text">
       <string>替换</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="replaceAll_btn">
      <property name="text">
       <string>全部替换</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>120</y>
     <width>239</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="findPrev_btn">
      <property name="text">
       <string>查找上一处</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="findNext_btn">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>查找下一处</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="close_btn">
      <property name="text">
       <string>关闭</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
